src/librustc_codegen_ssa/back/linker.rs

```diff
@@ -1177,7 +1177,7 @@ impl<'a> Linker for WasmLd<'a> {
         }
 
         // LLD will hide these otherwise-internal symbols since our `--export`
-        // list above is a whitelist of what to export. Various bits and pieces
+        // list above is an allowlist of what to export. Various bits and pieces
```

Comment:
```
        // LLD will hide these otherwise-internal symbols since it only exports
        // symbols explicity passed via the `--export` flags above and hides all
        // others. Various bits and pieces
```